src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -8,19 +8,36 @@ import { useTabs, useDefaultPath } from "hooks";
 import { CodeChanges } from "pages/patch/patchTabs/CodeChanges";
```

Comment:
nit. i know this wasnt you but we should start setting some patterns around how we import external deps vs internal ones. How about External first, Then internal and everything in alphabetical order. Im sure theres probably an eslint extension that will automate this for us. what do you think?

Reply:
sorting imports is a divisive topics in the js community. some devs care a lot, some don't care at all. i struggle to see the value in sorting imports.

Reply:
Although the order doesn't make a difference I think it just produces cleaner looking code and it's easy to look at the deps and see what you have and what you don't. Same with sorting components and styles it's nice to be able to quickly glance and see what you have and don't have.

Reply:
if it can be automated by a tool like prettier then i'm all for it